code/game/gamemodes/blob/blobs/node.dm

```diff
@@ -46,7 +46,10 @@
 			M.playsound_local(loc, adminblob_beat, 50, 0, null, FALLOFF_SOUNDS, 0)
 
 	for(var/i = 1; i < 8; i += i)
-		Pulse(5, i)
+		if(overmind)
+			Pulse(5, i, overmind)
```

Comment:
Surely you could just have Pulse(5, i, overmind) rather than the checks, as it's okay if overmind is null